python/ray/tune/execution/ray_trial_executor.py

```diff
@@ -56,6 +56,10 @@
     COPY_DIRECTORY_CHECKPOINTS_INSTEAD_OF_MOVING_ENV,
     "TUNE_CHECKPOINT_CLOUD_RETRY_NUM",
     "TUNE_CHECKPOINT_CLOUD_RETRY_WAIT_TIME_S",
+    "AWS_ACCESS_KEY_ID",
+    "AWS_SECRET_ACCESS_KEY",
+    "AWS_SECURITY_TOKEN",
+    "AWS_SESSION_TOKEN",
```

Comment:
Without this, the pyarrow test complains, and the fsspec test fails because no credentials are found in the trainable workers.

Maybe we want to make these variables configurable at some point. But I think propagating these vars is actually desired behavior.